database/migrations/2021_05_20_200000_add_language_Code_to_streams_table.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddLanguageCodeToStreamsTable extends Migration
+{
+    public function up()
+    {
+        Schema::table('streams', function (Blueprint $table) {
+            $table->string('language_code')->default('en');
```

Comment:
Please add the new field after `status`.

Reply:
Fixed!

Reply:
I believe it was changed again by the merge?